README.md

```diff
@@ -1,2 +1,107 @@
+
+This repository uses [*Lerna*](https://github.com/lerna/lerna) to manage multiple [*npm*](https://www.npmjs.com/) packages. We 
+configured Lerna to use [*Yarn Workspaces*](https://yarnpkg.com/lang/en/docs/workspaces/) under the hood.
+
+# Installation
+
+Refer to individual packages' *Installation* section in order to learn more about how you can use each package.
+There is no top-level installation. Though, [`eslint-config-udemy-basics`](packages/eslint-config-udemy-basics)
+and [`eslint-plugin-udemy`](packages/eslint-plugin-udemy) are good starting points.
+
+# Commands
+
+In order to contribute to this repository, you need to have a basic understanding of 
+[Lerna commands](https://github.com/lerna/lerna#commands). In a nutshell, 
+there are three important commands to know:
+
+* **[`lerna add`](https://github.com/lerna/lerna#add):** In order to add a new dependency to an existing package's 
+`package.json` file, you'd run `` `yarn bin`/lerna add dependency-name --scope=package-name``. e.g.
+    ```
+    $ `yarn bin`/lerna add eslint-plugin-lodash --scope=eslint-config-udemy-website
+    ``` 
+* **[`lerna bootstrap`](https://github.com/lerna/lerna#bootstrap):** In order to install all of the dependencies
+for every package, you can simply run `` `yarn bin`/lerna bootstrap``
+* **[`lerna publish`](https://github.com/lerna/lerna#publish):** In order to publish your changes to npm, you
+can simply run `` `yarn bin`/lerna publish ``. It would prompt you to decide version numbering for each changed
+package.
+
+# List of available packages
+
+We individually publish all folders inside [`packages`](packages) as a separate npm package. These packages are
+owned by our [Udemy npm organization](https://www.npmjs.com/org/udemy).
+
+* **[`eslint-config-udemy-basics`](packages/eslint-config-udemy-basics):** A basic ESLint configuration for writing 
+ES2015 JS code. Used by various Udemy projects.
+* **[`eslint-config-udemy-jasmine-addons`](packages/eslint-config-udemy-jasmine-addons):** 
+[*Jasmine*](https://github.com/jasmine/jasmine) specific ESLint configuration. Extended by 
+[`eslint-config-udemy-website`](packages/eslint-config-udemy-website).
+* **[`eslint-config-udemy-react-addons`](packages/eslint-config-udemy-react-addons):**
+[*React*](https://github.com/facebook/react) specific ESLint configuration. Extended by 
+[`eslint-config-udemy-website`](packages/eslint-config-udemy-website).
+* **[`eslint-config-udemy-website`](packages/eslint-config-udemy-website):**
+The ESLint configuration used by 
+[`udemy/website-django/static/`](https://github.com/udemy/website-django/tree/master/static/.eslintrc.js) codebase.
+* **[`eslint-plugin-udemy`](packages/eslint-plugin-udemy):**
+A set of custom ESLint rules written for Udemy, by Udemy. This rules are mainly used by  
+[`eslint-config-udemy-website`](packages/eslint-config-udemy-website).
+
+# Contributing
+
+Install [*Yarn@1.3.2*](https://github.com/yarnpkg/yarn/releases/tag/v1.3.2) globally. 
```

Comment:
Lowercase Yarn?

Reply:
Are you sure you want to tell the user how to install Yarn since different people have different ideas on how this should be done?

Reply:
Hmm, you have a point. I still think yarn is wrong about insisting on discouraging installation through npm. I'd like to keep this npm based approach here since it is related with the "note" I put below it.